apps/dashboard/app/(app)/apis/create-api-button.tsx

```diff
@@ -38,7 +38,9 @@ export const CreateApiButton = ({ ...rest }: React.ButtonHTMLAttributes<HTMLButt
     },
     onError(err) {
       console.error(err);
-      toast.error(err.message);
+      let temp = JSON.parse(err.message);
```

Comment:
You can turn this into a util and parse the message across versus copy and pasting the same code. 

then it would be something like

```
const message = errorhandler(err);
toast.error(message)
```

Where you error handler would handle your error parsing.